includes/class-freemius.php

```diff
@@ -6450,6 +6480,196 @@ function is_themes_page() {
 		}
 
 		#----------------------------------------------------------------------------------
+		#region Affiliation
+		#----------------------------------------------------------------------------------
+
+        /**
+         * @author Leo Fajardo
+         * @since 1.2.1.7.2
+         *
+         * @return bool
+         */
+        function has_affiliation() {
+            if ( ! is_object( $this->_plugin ) ) {
+                return false;
+            }
+
+		    return $this->_plugin->has_affiliation();
+        }
+
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 1.2.1.7.2
+         */
+        private function load_affiliate_and_terms() {
```

Comment:
@fajardoleo for consistency, change it to `fetch_affiliate_and_terms()`. `load_` is when things are loaded from the storage while `fetch_` is when the data fetched from an API.

Reply:
👍